client/scss/components/_help-block.scss

```diff
@@ -16,7 +16,9 @@
   }
 
   a {
-    color: $color-link;
+    color: theme('colors.secondary.DEFAULT');
+    text-decoration: 2px solid underline;
```

Comment:
just a reminder - this will not work for safari https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration
Also - the underline should hide on hover

Not huge - will fix up.

Reply:
I didn't add backup for safari ?🤔